### DISCUSSION TITLE
Practica HTML

### DISCUSSION THREAD

**Discussion by AndresAVargas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#222] [Q&A]
Compañeros una duda, para la Maquetación de la pagina del cliente y la del pastelero, todo va en un archivo HTML o son dos archivos distintos?

**Comment by AngelCruzO** [comment]
Respuesta corta, pueden ser dos. 

Pero el número de archivos HTML, depende de cómo piensas tú qué debería de ir. Puede ser una para tener todo ahí; dos para separar usuarios y pastelero; tres usuarios, pedidos y pastelero. El límite es como deseas que la experiencia se usuario sea mejor

**Reply by AndresAVargas** [reply]
Disculpa podrias explicarme como hacer la pagina del pastelero, es que ahí si estoy perdido, desarrolle la del cliente y la del pastelero solo me falta esa y no se como desarrollarla D:

**Comment by carlos-gmx3** [comment]
2, ya que normalmente un panel de administración y el área de clientes se manejan de forma independiente en sitios propios, incluso en un marketplace como amazon hay paginas dedicadas a administradores para atención al cliente o cosas así.
Puedes hacer una pagina principal, una donde llenen el formulario para el pedido y por último donde el pastelero ve las ordenes, en esto úñtimo bien puedes inventar los datos para que se muestren de forma fija

**Reply by AndresAVargas** [reply]
Disculpa podrias explicarme como hacer la pagina del pastelero, es que ahí si estoy perdido, desarrolle la del cliente y la del pastelero solo me falta esa y no se como desarrollarla x2

**Reply by RocioGH** [reply]
Yo también tengo duda con la parte del pastelero, solo entiendo que hay que hacer una tabla, pero eso de las existencias de sabores y adornos no se como hacerlo

**Reply by JorgeLReyes** [reply]
> Disculpa podrias explicarme como hacer la pagina del pastelero, es que ahí si estoy perdido, desarrolle la del cliente y la del pastelero solo me falta esa y no se como desarrollarla x2

Puedes agregarle una tabla para que tenga los pedidos ejemplo: nombre del cliente, telefono, email, numero de pedido, fecha de entrega, etc.
Otra donde tenga un inventario de sus ingredientes, etc.
En este caso como solo es maqueta podrias rellenarla con datos falsos solo para simular como seria.

**Reply by carlos-gmx3** [reply]
Exacto, inventa los datos de cómo se vería
Aún sabiendo javascript necesitas de un backend que procese la información
